packages/now-cli/test/helpers/prepare.js

```diff
@@ -252,6 +252,11 @@ module.exports = (req, res) => {
         },
       }),
     },
+    'failing-alias': {
+      'now.json': JSON.stringify(
+        Object.assign(JSON.parse(getConfigFile(true)), { alias: 'zeit.co' })
```

Comment:
Perhaps you should use something other than zeit.co in case the tests somehow run as a zeit account and screw up the home page 😮 

```suggestion
        Object.assign(JSON.parse(getConfigFile(true)), { alias: 'example.com' })
```

Reply:
Also, the test is failing:

> ✖ test "now.json" aliasing with non-owned domain

Reply:
@styfle It need to be a domain added to the platform though, otherwise we display a different error